modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/index/CompanyIndexFactoryTest.java

```diff
@@ -249,6 +249,23 @@ public void testDefaultIndices() throws Exception {
 		_assertMappings(Field.COMPANY_ID, Field.ENTRY_CLASS_NAME);
 	}
 
+	@Test
+	public void testExecuteIndexContributorOnBeforeRemove() throws Exception {
+		IndexContributor indexContributor = Mockito.mock(
+			IndexContributor.class);
+
+		addIndexContributor(indexContributor);
+
+		createIndices();
+		deleteIndices();
+
+		Mockito.verify(
+			indexContributor, Mockito.times(1)
+		).onBeforeRemove(
```

Comment:
should also assert onAfterCreate is called 1 time?

but i was actually thinking we test this at the CompanyIndexFactoryHelper level, seems like testIndexContributors is already testing this at the CompanyIndexFactory level.

also, maybe testIndexContributorsThrowsException is too, but i dont actually understand that test and if its really able to assert if something were to break